.github/workflows/cmake.yaml

```diff
@@ -2,7 +2,6 @@ name: CMake
 
 on:
   push:
-    branches: [ $default-branch ]
```

Comment:
Why change it?

Reply:
Because need run github action in my clone.

Reply:
Let's remove it then and submit.Thanks!